jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ContainerConfigurationTemplate.java

```diff
@@ -152,6 +184,41 @@ public void setContainerCmd(@Nullable List<String> cmd) {
     config.Cmd = cmd;
   }
 
+  public void setContainerHealthCheckTest(@Nullable List<String> test) {
+    if (config.Healthcheck == null) {
+      config.Healthcheck = new HealthCheckObjectTemplate();
+    }
+    Preconditions.checkNotNull(config.Healthcheck).Test = test;
```

Comment:
Do we need these `checkNotNull`s? You already checked it above.

Reply:
I think NullAway only checks the control flow, so it still gives me a compile error because I don't return after the null check (it doesn't realize `Healthcheck` is being instantiated if it's null).